scripts/fao/preprocess.py

```diff
@@ -0,0 +1,87 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#         https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""A script that takes in thr .csv file and cleans the data, outputting a
+new .csv file named "output.csv".
+"""
+import csv
+import pycountry
+
+INPUT_CSV_FILENAME = 'CurrencyFAO.csv'
+OUTPUT_CSV_FILENAME = 'cleaned_output.csv'
```

Comment:
nit: could we produce the output in its own directory? So, have a folder called "output/" and then have the output file go there? That's who a bunch of other import scripts work too.

Reply:
done!